api/src/main/java/org/ihtsdo/rvf/messaging/ValidationQueueManager.java

```diff
@@ -64,6 +68,9 @@ public void queueValidationRequest(ValidationRunConfig config, Map<String, Strin
 				String configJson = gson.toJson(config);
 				LOGGER.info("Send Jms message to queue for validation config json:" + configJson);
 				jmsTemplate.convertAndSend(destinationName, configJson);
+				messagingHelper.send(config.getResponseQueue(),
```

Comment:
You need to check that response queue is not null first before sending response message. (i.e sending response messages only when the queue is specified).

Reply:
Fixed.